.github/workflows/draft-release.yml

```diff
@@ -69,9 +69,9 @@ jobs:
           node-version: 14
 
       - name: Setup Go
-        uses: actions/setup-go@v4
+        uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.1
```

Comment:
Could we pin to @v5 instead of the actual hash? The setup-go docs seem to reference the major version, so I would assume they would want to keep things mostly compatible within the same major version (so we wouldn't get issues with the action being updated under us, but instead we could benefit from things like security updates)